src/components/Visualization.js

```diff
@@ -4,32 +4,33 @@ import LineChart from './LineChart'
 import CaVsNy from './CaVsNy'
 
 
-/* followed a tutorial for de-structuring and then rebuilding the object in Chart 
-was a good way to get familiar with API fetching and chartjs 
-but ran into a lot of errors so decided to implement a new 
+/* followed a tutorial for de-structuring and then rebuilding the object in Chart
+was a good way to get familiar with API fetching and chartjs
+but ran into a lot of errors so decided to implement a new
 method that made more sense to me (LineChart.js and CaVsNy.js) */
 
-function Visualization(){
+function Visualization() {
     const [stateData, setStateData] = useState([])
     const url = "https://covidtracking.com/api/v1/us/current.json"
 
-  useEffect(() => {
     async function getData() {
-      const res = await fetch(url);
```

Comment:
I would make sure to use consistent spacing and either stick to 2 or 4 spaces - we use 4 spaces